### PR TITLE
implement fourth.SaveAvatar

### PR DESCRIPTION
**Pull request by HumanoidSandvichDispenser**
fetch request in stealAvatarButton.js is moved to a fourth method fourth.SaveAvatar to avoid making arbitrary fetch requests https://github.com/aprzn123/TheThirdCan/issues/22#issuecomment-1457212864 and also avoid rewriting code for #38

### CODE REVIEW

File: src/injected/stealAvatarButton.js [hunk 1]
```diff
@@ -22,24 +22,11 @@
 
   function stealAvatar() {
     const img = document.querySelector("#content_host div > img");
-    console.log(img);
     const data = drawImageToNewCanvas(img);
+    // there should be a way to cache the user's id so we do not have to pass an `id` parameter
+    // to API functions
```
Comment (HumanoidSandvichDispenser) [inline src/injected/stealAvatarButton.js:27]: in retrospect i think this isn't worth the hassle

Comment (HumanoidSandvichDispenser) [reply src/injected/stealAvatarButton.js:27]: unless there is some way that the extension is going to track the user's id across pages

**Review by aprzn123 — CHANGES_REQUESTED** [review]
can you document the new function in `docs/third_fourth.md`? we don't have doc gen from the comments atm

**Comment by thekifake** [comment]
> we don't have doc gen from the comments atm

Hopefully we can do something about that eventually. I'll see if I can maybe figure things out.

**Review by aprzn123 — APPROVED** [review]
i feel like i've been avoiding merging this but tbh i really don't know why so let's just go for it lol

**Pull request merged**